runtime/integration-tests/src/generic/envs/runtime_env.rs

```diff
@@ -101,7 +116,9 @@ impl<T: Runtime> Env<T> for RuntimeEnv<T> {
 				}
 				_ => None,
 			})
-			.unwrap();
+			// NOTE: This is actually not always correct. Even if there is not fee event, there is a
+			//       fee substracted if the `pre_dispatch()` errors out.
```

Comment:
Is there any way to get that value from `pre_dispatch()`?

Reply:
You mean the fee amount? do not think so.